joomla/joomla-cms
# reCAPTCHA V1 stops on March 31

Google have emailed directly anyone using v1 reCAPTCHA keys but users don't read their email.

This PR adds a post-installation message IF they have the reCAPTCHA plugin enabled AND they are using v1 keys.

This PR also updates the messages in the plugin informing them that V1 will not work after march 31

@mbabker already completely refactored the plugin for J4 to remove V1 etc this PR is ust for the messages

### Test Instructions
1. Apply this PR
2. Update the db with the following sql
```
INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description_key`, `action_key`, `language_extension`, `language_client_id`, `type`, `action_file`, `action`, `condition_file`, `condition_method`, `version_introduced`, `enabled`)
VALUES
(700, 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_TITLE', 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_BODY', 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_ACTION', 'plg_captcha_recaptcha', 1, 'action', 'site://plugins/captcha/recaptcha/postinstall/actions.php', 'recaptcha_postinstall_action', 'site://plugins/captcha/recaptcha/postinstall/actions.php', 'recaptcha_postinstall_condition', '3.9.0', 1);
```
3. Check the Post-intallation messages in the following scenario
3.1 recaptcha plugin disabled => no message
3.2 recaptcha plugin enabled with V2 => no message
3.3 recaptcha plugin enabled with V1 => message and link
NOTE you can enter anything for the keys
